ui/Screen/Discovery.svelte

```diff
@@ -0,0 +1,43 @@
+<script>
+  import { getFeed } from "../src/discovery.ts";
+
+  import { SidebarLayout } from "../DesignSystem/Component";
+  import { Icon, Input, Title } from "../DesignSystem/Primitive";
+
+  import Project from "./Discovery/Project.svelte";
+
+  const projectFeed = getFeed();
+</script>
+
+<style>
+  .container {
+    margin: 64px 100px;
+  }
+
+  .projects {
+    display: grid;
+    grid-template-columns: minmax(0, 1fr) minmax(0, 1fr);
+    column-gap: 32px;
+    row-gap: 32px;
+  }
+</style>
+
+<SidebarLayout>
+  <div class="container">
+    <Title variant="huge" style="margin-bottom: 28px;">Discover</Title>
```

Comment:
we try to use multiples of `8px` for everything in the design file. So I'd try `32px` here. (just messy design file)